using_redcap_api/using_redcap_api.md

```diff
@@ -0,0 +1,436 @@
+
+This is an example of trying out and learning about the API without having to write code. You can experiment with various methods and learn easily without having to go through the frustration of writing a lot of code to find out how to do things.
+
+In some API methods (like "Export Records"), you'll have a number of drop down menus you can choose from, including which "forms" and which "fields" you want to retrieve. You can make multiple choices in menus like these by holding down control while clicking (Windows) or command while clicking (Mac).  Helpfully, however, if you want **every** item listed (say, you want every form and every field), you don't click any selection at all, and REDCap assumes you mean everything that appears in the drop-down list.
+
+
+### API Playground: Code
+
+So, you've experimented in the API Playground and you know the kind of API call you want to make. Now, how do you automate this selection so that your R or Python code will issue exactly the same request using code?  REDCap's API playground comes to the rescue again, with code snippets.
+
+In the tabs at the bottom of the API Playground, you'll see names of programming languages, including Python and R.  Below you can see what the Python code looks like for the API request to retrieve all records from the cervical cancer database:
+
+![Python code from the API playground](media/python_code.png)<!-- style = "border :1px solid rgb(var(--color-highlight));" -->
+
+We'll show you how to use Python and R in two separate sections, so you can feel free to look at either one, or both!
+
+## Python and the REDCap API
+
+<div class = "important">
+There is a useful Python library that makes importing REDCap data into Python simpler than the code REDCap suggests.  Nevertheless, we think it's important to show you how to use the code REDCap provides, because it is reliable and reflects the current state of the REDCap API.  
+
+The Python library, PyCap, that streamlines this effort is great and seems very stable, and we provide a link to it toward the end of this module, but Python libraries depend on volunteers who keep them updated.  Therefore, we think it's useful to know how to use the basic method we'll share here, even if in the future you prefer a different approach.
+</div>
+
+Open a Jupyter notebook (running Python 3) and paste the code from the API Playground's "Python" tab into your first cell and run that cell.  It works!  Kind of.  Take a look below to see what the result of our API call for the records of cervical cancer data looks like.
+
+![The same python code as before, now in a Jupyter notebook](media/jupyter_1.png)<!-- style = "border :1px solid rgb(var(--color-highlight));" -->
+
+What's the only problem with this?  Well, it's not in a pandas data frame, which is what we'd prefer to work with.  We can fix that by adding a few lines of code!  Add this to a second cell, and run it.
```

Comment:
Add a reminder/help about collapsing the output by clicking to the left of the output box.

Reply:
I'm not sure what you're referring to here @drelliche ?  Are you talking about resizing something in a Jupyter notebook?

Reply:
@pm0kjp Yes! I was talking about resizing something in a Jupyter notebook! Sorry for my vague comment. 

But on closer inspection of different places someone might be following along, I am no longer so sure this is necessary at all. I was in a Jupyter Lab when I wrote this and it seems less likely that learners will end up there.